platform/openide.util/test/unit/src/org/openide/util/BaseUtilitiesTest.java

```diff
@@ -126,8 +206,10 @@ public void testFileURI() throws Exception {
             assertEquals(new File("/some/path"), BaseUtilities.toFile(new URI("file:///some/path")));
             assertEquals(new File("/some/path"), BaseUtilities.toFile(new URI("file:/some/path/")));
         }
+        //the following code seems not to have sense at all
```

Comment:
Because it tests URI class and not toURI method - even though it call it. 
It seems to be a test of JVM implementation itself - I have no idea what for.

Reply:
Again. this comment in the PR, so reviewers undestand what you're trying to do. Thanks!.

Reply:
i think it wants to test the method `BaseUtilities.toURI()` against expected results taken by using URI directly. It doesn't look too bad in my eyes.